### PR TITLE
Fixes wires not working in map elements (vaults/away missions)

### PR DESCRIPTION
**Pull request by unid15**

### CODE REVIEW

File: code/datums/map_elements.dm [hunk 1]
```diff
@@ -23,6 +23,10 @@ var/list/datum/map_element/map_elements = list()
 	if(!location && objects.len)
 		location = locate(/turf) in objects
 
+	//Build powernets
+	for(var/obj/structure/cable/C in objects)
+		C.rebuild_from()
```
Comment (PJB3005) [inline code/datums/map_elements.dm:31]: Probably a bad idea to call `rebuild_from` if it already has a power net

Comment (unid15) [reply code/datums/map_elements.dm:31]: Map loader-spawned cables don't have power nets though

Comment (PJB3005) [reply code/datums/map_elements.dm:31]: Yes, but once a cable gets a power net built, all adjacent cables do too so calling the proc is a waste of cpu time of the already have a power net.

**Review by PJB3005 — APPROVED** [review]

File: code/datums/map_elements.dm (hunk 1 above)
Comment (gbasood) [reply code/datums/map_elements.dm:31]: Weird that reviews show up AFTER new commits show up

**Pull request merged**